README.md

```diff
@@ -1,3 +1,49 @@
 # azure-file-csi-driver-operator
 
 An operator to deploy the [Azure File CSI Driver](https://github.com/openshift/azure-file-csi-driver) in OKD.
+
+This operator is installed by the [cluster-storage-operator](https://github.com/openshift/cluster-storage-operator).
+
+
+
+origin-cluster-cloud-controller-manager-operator
+
+
+# Quick start
+
+Before running the operator manually, you must remove the operator installed by CSO/CVO
+
+```shell
+# Scale down CVO and CSO
+oc scale --replicas=0 deploy/cluster-version-operator -n openshift-cluster-version
+oc scale --replicas=0 deploy/cluster-storage-operator -n openshift-cluster-storage-operator
+
+# Delete operator resources (daemonset, deployments)
+oc -n openshift-cluster-csi-drivers delete deployment.apps/azure-file-csi-driver-operator deployment.apps/azure-file-csi-driver-controller daemonset.apps/azure-file-csi-driver-node
+```
+
+To build and run the operator locally:
+
+```shell
+# Build the operator
+make
+
+# Set kubeconfig and obtain desired version
+export KUBECONFIG=<path-to-kubeconfig>
+export OPERATOR_IMAGE_VERSION=$(oc get clusterversion/version -o json | jq '.status.desired.version' | tr -d "\"")
```

Comment:
I slightly prefer `jq -r` instead of the whole `tr`. It will un-escape any other weird characters.

Reply:
We already use `tr` in https://github.com/openshift/azure-disk-csi-driver-operator/pull/94/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R28

/lgtm
/approve

Reply:
That seems better indeed, I updated to `jq -r` in both repos: https://github.com/openshift/azure-disk-csi-driver-operator/pull/99